examples/external_models/alpenglow/prepare_data.py

```diff
@@ -0,0 +1,15 @@
+import pandas as pd
+import sys
+from alpenglow.experiments import ExternalModelExperiment
+import pathlib
+
+pathlib.Path("./batches").mkdir(parents=True, exist_ok=True)
+
+data = pd.read_csv('/mnt/idms/kdomokos/data/tutorial_data/data.csv', header=None, names=['time', 'user', 'item'])
```

Comment:
Ez lehetne egy webcím az idms path helyett. Az adatot felraktam az infóra, remélem, a kódjavaslatot nem szúrtam el :-)
```suggestion
data = pd.read_csv('http://info.ilab.sztaki.hu/~fbobee/alpenglow/tutorial_dataset.csv', header=None, names=['time', 'user', 'item'])
```